extensions/amp-animation/0.1/web-animations.js

```diff
@@ -895,6 +925,34 @@ class CssContextImpl {
     return this.getElementSize_(this.requireTarget_());
   }
 
+  /** @override */
+  getElementSize(selector, selectionMethod) {
+    return this.getElementSize_(this.getElement_(selector, selectionMethod));
+  }
+
+  /**
+   * @param {string} selector
+   * @param {?string} selectionMethod
+   * @return {!Element}
+   * @private
+   */
+  getElement_(selector, selectionMethod) {
+    dev().assert(
+        selectionMethod == null || selectionMethod == 'closest',
+        'Unknown selection method: %s', selectionMethod);
+    let element;
+    try {
+      if (selectionMethod == 'closest') {
```

Comment:
Can we have an enum-like constant for this? Mainly to document the different methods and how they behave.